Dockerfile

```diff
@@ -1,4 +1,4 @@
-ARG DOCKER_GEN_VERSION
+ARG DOCKER_GEN_VERSION=latest
```

Comment:
Look like I forgot to set a default, thanks.

Can you please set it to `0.9.0` instead of `latest`?
this way we are building our image with a fixed (and working) dependency